src/hyperion/experiment_plans/rotation_scan_plan.py

```diff
@@ -116,7 +114,7 @@ def calculate_motion_profile(
     See https://github.com/DiamondLightSource/hyperion/wiki/rotation-scan-geometry
     for a simple pictorial explanation."""
 
-    direction = ROTATION_DIRECTION[params.rotation_direction]
```

Comment:
In general I agree, but this syntax basically exists exactly for sequences of calculations like this, and I like it here, it's the only thing in the whole function, there's nothing mysterious about it, just  takes half the space of doing it separately.

Anyway it's not part of this PR :P

Reply:
https://github.com/DiamondLightSource/hyperion/pull/1473/commits/f06978248aaefcf3ba44885cfa7e80c0f0a66c9f

To separate the calculation and logging is 35 more lines, in my opinion way less readable. It requires you to make any changes 2 or 3 times and therefore and has a risk that the logging doesn't accurately describe the calculation, while the original way guarantees that the exact calculation is what is logged. But since you both feel this way we can do this instead.

Reply:
Nit: It would probably be more readable in the logs to do something like: `LOGGER.info("scan_width_deg ({scan_width_deg}) = num_images ({num_images}) * params.rotation_increment_deg ({params.rotation_increment_deg})")`